components/kyma-environment-broker/internal/process/deprovisioning/remove_runtime_test.go

```diff
@@ -32,50 +33,16 @@ func TestRemoveRuntimeStep_Run(t *testing.T) {
 
 		// when
 		entry := log.WithFields(logrus.Fields{"step": "TEST"})
-		result, repeat, err := step.Run(operation, entry)
+		result, repeat, err := step.Run(operation.Operation, entry)
 
 		// then
 		assert.NoError(t, err)
-		assert.Equal(t, 1*time.Second, repeat)
+		assert.Equal(t, 0*time.Second, repeat)
```

Comment:
the old implementation of that step returns 1 second if everything was ok, then the check if provisioner finished was done in the initialisation step. This PR makes it simpler